pkgs/elastic-dashboard/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  flutter,
+  fetchFromGitHub,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+flutter.buildFlutterApplication rec {
+  pname = "elastic-dashboard";
+  version = "2024.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Gold872";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-u8pqkaChSD01DBdbn43txIQBvfRQ8lLmeT7rbGdVJcE=";
+  };
+
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  nativeBuildInputs = [copyDesktopItems];
+
+  # Get everything out of $out/app to avoid conflicts with other flutter packages
+  postInstall = ''
+    mkdir "$out"/opt
+    mv "$out"/app "$out"/opt/${pname}
+    ln -sf "$out"/opt/${pname}/elastic_dashboard "$out"/bin/elastic_dashboard
+    install -Dm444 ${src}/assets/logos/logo.png "$out"/share/pixmaps/${pname}.png
+  '';
+
+  desktopItems = [
+    (makeDesktopItem {
+      desktopName = "Elastic";
+      name = pname;
+      exec = "elastic_dashboard";
+      icon = pname;
+      comment = meta.description;
+      categories = ["Development"];
+      keywords = ["FRC" "Dashboard"];
+    })
+  ];
+
+  meta = with lib; {
+    mainProgram = "elastic_dashboard";
+    description = "A simple and modern dashboard for FRC ";
+    homepage = "https://github.com/Gold872/elastic-dashboard";
+    license = licenses.mit;
+    maintainers = with maintainers; [max-niederman];
```

Comment:
I'm just going to remove the maintainers list entirely for the time being since you're not in `lib.maintainers`. It didn't error when you added `arch-b1w` because you never actually evaluated `meta.maintainers` (remember Nix is lazy).